get_all_manifests.sh

```diff
@@ -54,16 +52,10 @@ rm -fr ./odh-manifests/* ./.odh-manifests-tmp/
 
 mkdir -p ./.odh-manifests-tmp/ ./odh-manifests/
 wget -q -c ${MANIFESTS_TARBALL_URL} -O - | tar -zxv -C ./.odh-manifests-tmp/ --strip-components 1 > /dev/null
+
 # mm-monitroing
 cp -r ./.odh-manifests-tmp/modelmesh-monitoring/ ./odh-manifests
-# modelmesh
-cp -r ./.odh-manifests-tmp/model-mesh/ ./odh-manifests
-cp -r ./.odh-manifests-tmp/odh-model-controller/ ./odh-manifests
-cp -r ./.odh-manifests-tmp/modelmesh-monitoring/ ./odh-manifests
-# Kserve
-cp -r ./.odh-manifests-tmp/kserve/ ./odh-manifests
-# workbench nbc
-cp -r ./.odh-manifests-tmp/odh-notebook-controller/ ./odh-manifests
+
```

Comment:
lets get this one removed as well.
i did the change in midstream but not clean up here. modelmesh-moniotirng is not needed in 2.5